<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSGB_EncoderGuiClass</class>
 <widget class="QMainWindow" name="OSGB_EncoderGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSGB Encoder Gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Map Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelJsonCount">
         <property name="text">
          <string>json count: 0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelB3dmCount">
         <property name="text">
          <string>b3dm count: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="importButton">
      <property name="statusTip">
       <string>Import Root Json</string>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Root Node</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="rootNodePath">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Root Json Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="exportButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string>Export files to dir</string>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Encode Key</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="encodeKeyEdit">
         <property name="placeholderText">
          <string>type key here...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="randomKeyButton">
         <property name="statusTip">
          <string>Generate random key</string>
         </property>
         <property name="text">
          <string>Generate Key Rondom </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="copyRandomKeyButton">
         <property name="statusTip">
          <string>Copy Encode Key</string>
         </property>
         <property name="text">
          <string>Copy Encode Key</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OSGB_EncoderGui.qrc"/>
 </resources>
 <connections/>
</ui>
